pkgs/games/fallout-ce/default.nix

```
@@ -0,0 +1,48 @@
+{ cmake
+, lib
+, SDL2
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation (finalAttr: {
+  pname = "fallout-ce";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "alexbatalov";
+    repo = "fallout1-ce";
+    rev = "v${finalAttr.version}";
+    hash = "sha256-EvRkOlvtiVao63S0WRKKuHlhfkdTgc0m6GTyv4EfJFU=";
+  };
+
+  fpattern = fetchFromGitHub {
+    owner = "Loadmaster";
+    repo = "fpattern";
+    rev = "v1.9";
+    hash = "sha256-/QvMQCmoocaXfDm3/c3IAPyfZqR6d7IiJ9UoFKZTpVI=";
+  };
+
+  buildInputs = [ cmake SDL2 ];
+  hardeningDisable = [ "format" ];
+  preConfigure = ''
+    substituteInPlace third_party/fpattern/CMakeLists.txt --replace "FetchContent_Populate" "#FetchContent_Populate"
+    substituteInPlace third_party/fpattern/CMakeLists.txt --replace "{fpattern_SOURCE_DIR}" "${finalAttr.fpattern}"
+    substituteInPlace third_party/fpattern/CMakeLists.txt --replace "$/nix/" "/nix/"
```

```suggestion
    substituteInPlace third_party/fpattern/CMakeLists.txt \
      --replace "FetchContent_Populate" "#FetchContent_Populate" \
      --replace "{fpattern_SOURCE_DIR}" "${finalAttr.fpattern}" \
      --replace "$/nix/" "/nix/"
```
please move this to postPatch